src/runtime/components/NuxtTurnstileNuxtScript.vue

```diff
@@ -0,0 +1,81 @@
+<script setup lang="ts">
+
+import { nextTick, useScriptCloudflareTurnstile, useRuntimeConfig, useNuxtApp, ref, onMounted, onBeforeUnmount } from '#imports'
+
+const props = defineProps({
+  // eslint-disable-next-line vue/require-default-prop
+  modelValue: {
+    type: String,
+    required: false,
+  },
+  element: {
+    type: String,
+    default: 'div',
+  },
+  // eslint-disable-next-line vue/require-default-prop
+  siteKey: {
+    type: String,
+    required: false,
+  },
+  options: {
+    type: Object as () => Omit<Partial<Turnstile.RenderParameters>, 'callback'>,
+    default: () => ({}),
+  },
+  resetInterval: {
+    type: Number,
+    default: 1000 * 250,
+  },
+})
+
+const emit = defineEmits<{
+  (name: 'update:modelValue', token: string): void
+}>()
+
+const config = useRuntimeConfig().public.turnstile
+const nuxtApp = useNuxtApp()
+
+const el = ref()
+const unmountStarted = ref(false)
+let id: string | undefined = undefined
+let interval: NodeJS.Timeout
+const { $script } = useScriptCloudflareTurnstile()
```

Comment:
We can just extract the `render` function from here and call it in onMounted safely, no need to use `load()`.